LCD.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * LCD.c
+ *
+ *  Created on: Feb 2, 2020
+ *      Author: helen
+ */
+
+#include "msp.h"
+#include "delay.h"
+#include "LCD.h"
+
+void init_LCD(void)                         //setup GPIO pins
+    {
```

Comment:
Remove the space for the line and the bracket. For example,
void init_LCD(void)
{
             (tab)bunch of code
}

Reply:
Got it!